<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>configWindow</class>
 <widget class="QMainWindow" name="configWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>config.toml</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextEdit_config">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>701</width>
      <height>591</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_saveconfig">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>590</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
